Yubico.YubiKey/src/Yubico/YubiKey/Piv/PivSessionExtensions.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2024 Yubico AB
+// 
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Globalization;
+
+namespace Yubico.YubiKey.Piv
+{
+    /// <summary>
+    /// Provides extension methods for the <see cref="PivSession"/> class, adding functionality
+    /// to check and handle unsupported algorithms for YubiKey devices.
```

Comment:
```suggestion
    /// This class provides extension methods for the <see cref="PivSession"/> class 
    /// to check and handle unsupported algorithms for YubiKey devices.
```

Reply:
Let me know if this wasn't the intended meaning.

Reply:
It was actually too specific so I updated it to be more allowing